packages/mdjs-core/src/mdjsStoryParse.js

```diff
@@ -61,11 +61,12 @@ function mdjsStoryParse({
   /* eslint-disable no-param-reassign */
 
   /**
-   * @param {UnistNode} node
+   * @param {UnistNode} _node
    * @param {number} index
    * @param {UnistParent} parent
    */
-  const nodeCodeVisitor = (node, index, parent) => {
+  const nodeCodeVisitor = (_node, index, parent) => {
+    let node = /** @type {UnistNode & {[key: string]: unknown}} */ (_node);
```

Comment:
Had to typecast this, even though UnistNode and Node both have `[key:string]:unknown` already in their interfaces, I still had to add it here.. Not sure why exactly >:(